ibeam/src/gateway_client.py

```diff
@@ -119,12 +136,12 @@ def new_chrome_driver(driver_path, headless: bool = True):
     options.add_argument('--no-sandbox')
     options.add_argument('--ignore-ssl-errors=yes')
     options.add_argument('--ignore-certificate-errors')
-    options.add_argument("--remote-debugging-port=9222")
+    #options.add_argument("--remote-debugging-port=9222")
```

Comment:
Hey, I'm wondering - why are you commenting this one out?

Reply:
Sorry I had to do this because when creating a second instance of web driver, that port was used.

Reply:
I forgot to resolve it properly.